.azure-pipelines-templates/release.yml

```diff
@@ -21,8 +21,45 @@ jobs:
           set -ex
           cd $(Build.ArtifactStagingDirectory)
           rename.ul + _ *+*.deb || true
+          ls
```

Comment:
Minor: Might be worth running `ls` _before_ the rename as well, so we can compare what was actually renamed?

Reply:
Sure, the rename isn't new and is thought to be fine, this was more of a pre-sign ls to see what we're feeding the signing, but why not.